jta-and-hibernate-standalone/src/main/java/org/jboss/narayana/quickstart/jta/EntityManagerProducer.java

```diff
@@ -14,18 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.narayana.quickstarts.jta.jpa;
+package org.jboss.narayana.quickstart.jta;
 
 import javax.enterprise.inject.Disposes;
 import javax.enterprise.inject.Produces;
 import javax.persistence.EntityManager;
 import javax.persistence.EntityManagerFactory;
 
+/**
+ * @author <a href="mailto:gytis@redhat.com">Gytis Trikleris</a>
+ */
```

Comment:
Normally acknowledgement of authorship goes in the README and is omitted from the source files to make it easier for users to cut and paste or copy the quickstarts into their own code.